File: challenge_two.py
s = 88

def squares(array):
    for i in range(len(array)):
        array[i] = array[i]*array[i]
    length = len(array)
    for i in range(length):
        for j in range(length):
            if (j+1 != length):
                if(array[j] > array[j+1]):
                    replace = array[j+1]
                    array[j+1] = array[j]
                    array[j] = replace
    result = list(filter(lambda item: item < s,array))
    return result
print("squares sorted in ascending order",squares([-6,-5,0,5,6])) 

File: challenge_two.js
function Order_squares(array){
    var limit = 88
    for (let i = 0; i < array.length; i++) {
        array[i] = Math.pow(array[i],2)
    }
    for(let i=0; i < array.length; i++){
        for(let j=0; j < array.length; j++){
            if(j+1 !== array.length){
                if(array[j] > array[j+1]){
                    let intercambio = array[j+1];
                    array[j+1] = array[j];
                    array[j] = intercambio;
                }
            }  
        }
    }
    let resultado = array.filter(element => element < limit)
    return resultado
}
console.log("squares sorted in ascending order",Order_squares([-8,-7,0,7,8]));